index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="https://fonts.googleapis.com/css?family=Inter:400,600,500,800" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style/style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- Welcome Header -->
+   <header class="header">
+       <h1>Welcome</h1>
+       <nav>
+           <i class="fa fa-bars" aria-hidden="true"></i>
+       </nav>
+   </header>
+
+   <!-- Main content -->
+    <section class="main1">
+        <div class="wrap-content">
+
+        <div class="intro">
+            <h5 class="name">Hey there. I'm Ntare Guy</h5>
```

Comment:
- [ ] Like the previous reviewer mentioned please make use of an `h1` tag instead of an `h5` tag. Your code should look like this `<h1 class="name">Hey there. I'm Ntare Guy</h1>` instead of this `<h5 class="name">Hey there. I'm Ntare Guy</h5>`